### PR TITLE
Class-06

### PR DESCRIPTION
**Pull request by Tenlia**
This is getting confusing, I'm working on sorting this out so that I can even apply what we learned in class. I will update this very soon.

### CODE REVIEW

**Comment by Tenlia** [comment]
Thanks to a LOT of help from Rick (Thanks!), I was able to think through how to alter the code in this one.

File: js/attach_articles.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+articlesRender = {};
+
+articlesRender.render = function() {
+	Entry.allEntries.forEach(function(entry) {
+    	$('#articles').append(entry.toHtml());
```
Comment (nhuntwalker) [inline js/attach_articles.js:5]: indentation.

File: js/make_articles.js [hunk 2]
```diff
@@ -24,14 +29,13 @@ Entry.prototype.toHtml = function() {
...
-});
-
-ourArticles.forEach(function(obj) {
-  allArticles.push(new Entry(obj));
-});
+Entry.processArticles = function(articlesPassedIn) {
+  articlesPassedIn.sort(function(a,b) {
+    return (new Date(b.pubDate)) - (new Date(a.pubDate));
+  })
+  .forEach(function(obj) {
```
Comment (nhuntwalker) [inline js/make_articles.js:36]: Better variable names than "obj" please. Perhaps something like "entryData"

File: js/view_sorting.js [hunk 3]
```diff
@@ -4,10 +4,10 @@ var viewSorting = {};
 
 viewSorting.fillFilters = function() {
   var aArray = [];
-  for(var o in ourArticles) {
-    for(var i = 0; i < ourArticles[o].authors.length; i++) {
-      if(!JSON.stringify(aArray).includes(ourArticles[o].authors[i])) {
-        aArray.push(ourArticles[o].authors[i]);
+  for(var o in Entry.allEntries) {
```
Comment (nhuntwalker) [inline js/view_sorting.js:7]: Definitely don't use this variable name. "o" could be mistaken for "0" easily.

**Pull request merged**